apps/src/aiTutor/views/legacyLabs/AiTutorSidebarSuggestedPrompts.tsx

```diff
@@ -28,7 +30,15 @@ const AiTutorSidebarSuggestedPrompts: React.FC<
             className={styles['ai-tutor-suggested-prompt-item']}
             aria-label={prompt.label}
             isIconOnly
-            icon={prompt.icon}
+            icon={
+              {
+                ...prompt.icon,
+                className: classNames({
+                  [styles['icon']]: true,
+                  [styles[`icon-${prompt.icon?.iconName}`]]: prompt.icon,
+                }),
```

Comment:
```suggestion
                className: classNames(styles['icon'], styles[`icon-${prompt.icon?.iconName}`]),
```
nit pick: I think this could be simplified a smidge by using the regular `classNames` arguments rather than object notation